scholia/app/views.py

```diff
@@ -134,6 +135,98 @@ def redirect_q(q):
     return redirect(url_for(method, q=q), code=302)
 
 
+def bioschemas_type(bioschemas_type, conforms_to):
```

Comment:
The following suggestion adds type annotations (you'll need to add `from typing import Any, Mapping` at the top, too)

```suggestion
def bioschemas_type(bioschemas_type: str, conforms_to: str) -> Mapping[str, Any]:
```

Reply:
I'm going to mark this one for future updates. I don't think this is generally done yet, and can be done in a more general code base upgrade.